tests/unit/test_session.py

```diff
@@ -56,8 +56,8 @@ def quoted(s):
         return f'"{s}"' if s else s
 
     assert (
-        str(Session(mock_server_connection))
-        == f"<snowflake.snowpark.session.Session: account={quoted(account)}, role={quoted(role)}, database={quoted(database)}, schema={quoted(schema)}, warehouse={quoted(warehouse)}>"
+            str(Session(mock_server_connection))
+            == f"<snowflake.snowpark.session.Session: account={quoted(account)}, role={quoted(role)}, database={quoted(database)}, schema={quoted(schema)}, warehouse={quoted(warehouse)}>"
```

Comment:
overall change looks good. There are some indentation changes I see across the file. Are they automatically added by linter?

Reply:
reverted